octopoes/octopoes/connector/octopoes.py

```diff
@@ -72,14 +73,20 @@ def list_objects(
         limit: int = DEFAULT_LIMIT,
         scan_level: set[ScanLevel] = DEFAULT_SCAN_LEVEL_FILTER,
         scan_profile_type: set[ScanProfileType] = DEFAULT_SCAN_PROFILE_TYPE_FILTER,
+        search_string: str | None = None,
+        order_by: Literal["scan_level", "object_type"] = "object_type",
+        asc_desc: Literal["asc", "desc"] = "asc",
     ) -> Paginated[OOIType]:
-        params: dict[str, str | int | list[str | int]] = {
+        params: dict[str, str | int | list[str | int] | None] = {
```

Comment:
This is incorrect usage; the `params` parameter of `httpx` methods is of type `QueryParamTypes`, which can't be a `None`. I suggest removing this type and explicitly filter out `None` values (just like we do in other usages of `httpx`)

Reply:
Like this?